### ISSUE TITLE
[BUG/ISSUE] integrate returned error

### ISSUE THREAD

**Issue by xpCH2** [geoschem/geos-chem#1660]
### What institution are you from?
South China University of Technology

### Description of the problem
I want to simulate with the full-chem model from 20210101 to 20211230. After the simulation runs for one day, reaction rates and integrate report an error (shown in GC_clean.log), and step size too smal (even T = 0.00 and H = 0.00). 

```
Forced exit from Rosenbrock due to the following error:
 --> Step size too small: T + 10*H = T or H < Roundoff
 T=   0.0000000000000000      and H=   0.0000000000000000     
 ### INTEGRATE RETURNED ERROR AT:           40          68           1
 Forced exit from Rosenbrock due to the following error:
 --> Step size too small: T + 10*H = T or H < Roundoff
 T=   0.0000000000000000      and H=   0.0000000000000000     
## INTEGRATE FAILED TWICE !!! 
 #############################################################################################################################################################################################################################################
 ### KPP DEBUG OUTPUT!
 ### Species concentrations at problem box           40          68           1
 #############################################################################################################################################################################################################################################
 ###                        NaN CH2I2
 ###                        NaN CH2IBr

...

 ###    0.0000000000000000      O2
 ###    0.0000000000000000      RCOOH
 #############################################################################################################################################################################################################################################
 ### KPP DEBUG OUTPUT!
 ### Reaction rates at problem box           40          68           1
 #############################################################################################################################################################################################################################################
                  Infinity SALAAL + O3 + SO2 --> LOx + PSO4 + SO4 - SALAAL
   1.8103838827844805E-010 SALAAL + HCl --> SALACL
   6.0635290677503539E-010 SALAAL + HNO3 --> LOx + NIT
                  Infinity SALCAL + O3 + SO2 --> SO4s + LOx - SALCAL
...
```

And if I add a reaction, it reported more error "integrate returned error" , but the step size H is about 10^-13, not 0.00 (shown in GC_add.log).

### Description of troubleshooting performed
I notice there are some "infinity" data in the reaction rates in KPP debug output, it seems like the issues #1547 and #1548, but it seems has not be solved. After I add a reaction, the error seems a little difference, may be adding a reaction can make a difference?

### GEOS-Chem version
GEOS-Chem 14.1.0 (released)

### Log files
- Run logs (if applicable): 
[GC_clean.log](https://github.com/geoschem/geos-chem/files/10785369/GC_clean.log)
[GC_add.log](https://github.com/geoschem/geos-chem/files/10785359/GC_add.log)

### Software versions
- CMake version: 3.13.2
- Compilers (Intel or GNU, and version): GNU 10.2.0
- NetCDF version: 4.9.0

**Comment by xpCH2** [comment]
If I am not interested in the problem reaction about `SALAAL` or `SALCAL`, can I delete these reaction in KPP files simply? I just want to simlute and analyse the produce of CH3COOH.

**Comment by yantosca** [comment]
Thanks for writing @xpCH2.  Have you added any modifications to the code?  If so we would recommend to try to build a clean version of GEOS-Chem Classic and then compare to your code.  Also see our [Debug GEOS-Chem and HEMCO errors](https://geos-chem.readthedocs.io/en/stable/geos-chem-shared-docs/supplemental-guides/debug-guide.html) documentation at geoschem.readthedocs.io

There have been a few other issues regarding this error.  You can look [here](https://github.com/geoschem/geos-chem/issues?q=is%3Aissue+integrate+return+error) to see the solutions that were offered.

You may also need to change the RTOL and ATOL values in fullchem_mod.F90 in order to allow the chemistry to converge.

NOTE: We are now asking users to identify themselves and their institutions when requesting help. Please do this in the future. Thanks!



NOTE: We are now asking users to identify themselves and their institutions when requesting help. Please do this in the future. Thanks!

**Comment by xpCH2** [comment]
@yantosca Thanks for the continued feedback and help. I didn't make any changes to the code.  I just deleted two reaction whose rate constant is infinity in fullchem.equand rebulid the mechanism. I'm not sure if this operator is reasonable, but the simulation can work successfully.

The deleted reacions:
```
SALAAL + O3 + SO2 --> LOx + PSO4 + SO4 - SALAAL
SALCAL + O3 + SO2 --> SO4s + LOx - SALCAL
```
In additon, I changed it (just for trying) into
```
SALAAL + O3 + SO2 --> LOx + PSO4 + SO4
SALCAL + O3 + SO2 --> SO4s + LOx 
```
but the problem didn't be solved. Maybe the sign isn't not the noly reason? I'm not sure but I hope this detail can help.

I am a student from South China University of Technology,
